Makefile

```diff
@@ -62,6 +66,16 @@ ifneq ($(OVERRIDE_IMAGE_NAME),)
 	sudo docker tag $(PREFIX)/$(DOCKER_IMAGE):$(VERSION) $(OVERRIDE_IMAGE_NAME)
 endif
 
+#This rule need to be run on Windows 19 server with golang, cygwin, mktemp and docker installed.
+container_win: build
+	cp $(OUT_DIR)/$(ARCH)/$(BINARY_NAME) $(TEMP_DIR)
+	cp deploy/docker/Dockerfile-win $(TEMP_DIR)/Dockerfile
+	docker build --pull -t $(PREFIX)/$(DOCKER_IMAGE):$(VERSION) $(TEMP_DIR)
```

Comment:
We should add a `-windows` suffix for these docker images.

Reply:
We are using manifest-tool to create a common name for all platform docker images